ocs_ci/ocs/ui/acm_ui.py

```diff
@@ -545,6 +555,53 @@ def create_cluster(self):
             self.deployment_status = "Creating"
             return
 
+    def add_different_pod_network(self):
```

Comment:
The way that the editor is being navigated here seems very prone to breaking, and difficult to troubleshoot if things go wrong during execution. Is there a better way to edit this network info? Either through `oc`or some other UI mechanism? Any change on the UI side could throw off the arrow/tab navigation here and cause this code to stop functioning.

Reply:
@prsurve , from our last discussion, any new ways in which we can edit this cluster yamls during OCP deploy through ACM?

Reply:
no other way we have to use yaml editor from acm ui

Reply:
> The way that the editor is being navigated here seems very prone to breaking, and difficult to troubleshoot if things go wrong during execution. Is there a better way to edit this network info? Either through `oc`or some other UI mechanism? Any change on the UI side could throw off the arrow/tab navigation here and cause this code to stop functioning.

@petr-balogh , could you please provide input here.

Reply:
@am-agrawa , I request your input here as well.

Reply:
So I had brought this up for discussion in DR sync-up some time back-
A couple of points which we discussed were-
1. I agree that the current implementation is highly prone to errors, but since it is working as of now (confirmed by Shylesh), and looking at the priority of this PR, it was acked by @hnallurv  to go ahead and merge it and later look for a better solution/implemnentation.
2. The reason why we are doing this is being we do not have direct locators to interact with the yaml editor which selenium needs.
3. This is a rare implementation and would require a lot of try and error methods to reach to a better solution which is highly time-consuming. We should surely prioritize it in near future.

Reply:
@OdedViner Can you have a quick sync-up with Shylesh and look into this implementation, please?

Reply:
My input was basically covered already by Aman :)  as there is no other solution and we don't know what else we can use we go with this approach for now.  The best way will be to just get the whole text from editor, delete it, load it to python object like ditcionary and then pretty print and put back to editor. But what I've heard from Shylesh last time even simply ctrl+a, ctrl+c doesn't work here to get simply all data from the editor.
Maybe we can bring up this to some UI developer from OCP who might suggest something?